packages/loot-core/src/server/crdt/timestamp.ts

```diff
@@ -315,24 +329,21 @@ Timestamp.since = isoString => {
  */
 Timestamp.DuplicateNodeError = class extends Error {
   constructor(node) {
-    super();
-    this.type = 'DuplicateNodeError';
-    this.message = 'duplicate node identifier ' + node;
+    super('duplicate node identifier ' + node);
+    this.name = 'DuplicateNodeError';
```

Comment:
I assumed we want to customise error name, not add a new custom property called type

Reply:
Makes sense!